src/kakarot/accounts/account_contract.cairo

```diff
@@ -295,6 +295,14 @@ func set_code_hash{syscall_ptr: felt*, pedersen_ptr: HashBuiltin*, range_check_p
     return ();
 }
 
+@external
+func get_authorized_pre_eip155_tx{syscall_ptr: felt*, pedersen_ptr: HashBuiltin*, range_check_ptr}(
```

Comment:
useless eventually right ?

Reply:
Yes
I still need to clean it
The PR is not ready for review